src-electron/zcl/zcl-loader-dotdot.js

```diff
@@ -818,7 +857,14 @@ async function processDataType(db, filePath, packageId, data, dataType) {
 function prepareNumber(a, dataType) {
   return {
     size: a.size,
-    is_signed: a.name.endsWith('u') || !a.name.includes('int') ? 0 : 1,
+    is_signed:
+      a.isSigned !== undefined
+        ? a.isSigned
+        : a.name.startsWith('u') ||
+            a.name.endsWith('u') ||
+            !a.name.includes('int')
+          ? 0
+          : 1,
```

Comment:
Do we do something similar on the zcl-loader-silabs?

Reply:
XMLs loaded through zcl-loader-silabs (Matter/Zigbee) expect the isSigned field to be specified in the XML. If not they do some regex matching - `/^int[0-9]{1,2}s?$/.test(a.$.name)`.